py-polars/polars/expr/expr.py

```diff
@@ -3867,10 +3867,8 @@ def filter(self, predicate: Expr) -> Self:
         ...     }
         ... )
         >>> df.group_by("group_col").agg(
-        ...     [
-        ...         pl.col("b").filter(pl.col("b") < 2).sum().alias("lt"),
-        ...         pl.col("b").filter(pl.col("b") >= 2).sum().alias("gte"),
-        ...     ]
+        ...     lt=pl.col("b").filter(pl.col("b") < 2).sum().alias("lt"),
+        ...     gte=pl.col("b").filter(pl.col("b") >= 2).sum().alias("gte"),
```

Comment:
Both keyword syntax and alias seems overkill :)

```suggestion
        ...     lt=pl.col("b").filter(pl.col("b") < 2).sum(),
        ...     gte=pl.col("b").filter(pl.col("b") >= 2).sum(),
```